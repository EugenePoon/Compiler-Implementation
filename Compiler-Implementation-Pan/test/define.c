int main()
{
    int a = 1;
    int b = 3.14;
    int c = 'p';
    print a;
    print b;
    print c;
}