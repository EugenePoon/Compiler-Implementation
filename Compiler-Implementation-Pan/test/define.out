24 19 0 5 25 13 0 1 12 13 0 1 1 26 1078523331 12 13 0 2 1 27 112 12 13 11 22 15 -1 13 0 1 1 11 22 15 -1 13 0 2 1 11 22 21 3